src/controller/proxy/controller.go

```diff
@@ -209,6 +236,19 @@ func (c *controller) putBlobToLocal(remoteRepo string, localRepo string, desc di
 }
 
 func (c *controller) waitAndPushManifest(ctx context.Context, remoteRepo string, man distribution.Manifest, art lib.ArtifactInfo, contType string, r RemoteInterface) {
+	if contType == manifestlist.MediaTypeManifestList {
+		_, payload, err := man.Payload()
+		if err != nil {
+			log.Errorf("failed to get payload, error %v", err)
+		}
+		dig := digest.FromBytes([]byte(payload))
+		key := getManifestListKey(art.Repository, string(dig))
+		log.Debugf("Cache manifest list with key=cache:%v", key)
```

Comment:
see my comment above, the log message is confusing with what's the key in cache.